File: Domain/Entities/ET/SkillMatrixMaster.cs
using System;
using System.Collections.Generic;

namespace Domain.Entities.ET;

public class SkillMatrixMaster : EntityBase
{
    public Guid Id { get; set; } = System.Guid.NewGuid();
    public string PositionCode { get; set; }
    public bool? Active { get; set; }
    public ICollection<SkillMatrixMasterDetail> SkillMatrixMasterDetails { get; set; } = new List<SkillMatrixMasterDetail>();
}

File: Web/ClientApp/src/app/models/et/skillMatrixMaster.ts
import { EntityBase } from "../entityBase";
import { SkillMatrixMasterDetail } from "./skillMatrixMasterDetail";

export class SkillMatrixMaster extends EntityBase {
    id: string;
    positionCode: string;
    active?: boolean;
    skillMatrixMasterDetails: SkillMatrixMasterDetail[] = []
}